tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
```

Comment:
@jaidisido How does this look to you?

This will create the following envs:

```
> tox -a
py37-standard
py37-distributed
py38-standard
py38-distributed
py39-standard
py39-distributed
py310-standard
py310-distributed
```

We would just need to update our test jobs to pass in `{standard || distributed}` in addition to python version. Example: `source $(poetry env info --path)/bin/activate && ./test.sh ${PY_ENV}-distributed ${NUM_THREADS}`